README.md

```diff
@@ -111,20 +104,24 @@ The [Product Management](/docs/articles/Products/ProductManagement.md) holds all
 
 ## Tutorials
 
-To start using the AbstractionLayer for your own projects you can jump into these tutorials:
+To start using the MORYX for your own projects you can jump into these tutorials:
```

Comment:
The first 3 tutorials are for the AbstractionLayer, the last one for Core. I wouldn't differentiate between them, because as a user you don't really care which one you're using